requirements/base.txt

```diff
@@ -6,13 +6,13 @@
 git+https://github.com/edx/XBlock.git@xblock-0.4.10#egg=XBlock==0.4.10
 
 # edx-submissions
-git+https://github.com/edx/edx-submissions.git@1.1.0#egg=edx-submissions==1.1.0
+git+https://github.com/edx/edx-submissions.git@efischer/gemnasium#egg=edx-submissions==1.1.0
```

Comment:
`edx-submissions` has the exact same alert and can be fixed exactly the same way, so I'm going to bundle the changes together.